Formula/racer.rb

```diff
@@ -0,0 +1,33 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+
+  url "https://github.com/phildawes/racer/archive/v1.1.0.tar.gz"
+  sha256 "f969e66d5119f544347e9f9424e83d739eef0c75811fa1a5c77e58df621e066d"
+  head "https://github.com/phildawes/racer.git"
+
+  depends_on "rust" => :build
+
+  resource "rust_source" do
+    url Formula["rust"].stable.url
```

Comment:
In that case, the `.fetch` approach looks more "honest".

However, I'm still be concerned about corner cases here though, because the resource is living with the "wrong" formula. So for example, if I install a `rust` `--HEAD` or `--devel` version, it would be best if racer used the sources for the version I actually have installed, not some random unrelated stable spec.